backend/src/api/base.clj

```diff
@@ -13,13 +13,21 @@
   (player-view/get-game-as-player (persistence/fetch-game game-id) player-id))
 
 (defn play-card-as-player
+  "Plays a card give and returns the game sate as seen by the player"
   [game-id player-id card-id row-id & target]
   (let [game-state (persistence/fetch-game game-id)]
+    (if (and (= game-id (:game-id game-state))
+          (or (= player-id (first (:player-ids game-state)))
+            (= player-id (second (:player-ids game-state)))))
```

Comment:
I'm pretty sure this can be done prettier... But how?

Reply:
I'll open an issue to clean this up cause it would involve changes to mocking, so I dont forget